command/src/main/java/com/jagrosh/jdautilities/command/Command.java

```diff
@@ -126,7 +126,7 @@
     
     /**
      * The aliases of the command, when calling a command these function identically to calling the
-     * {@link com.jagrosh.jdautilities.command.Command#name Command.name}.
+     * {@link Command#name Command.name}.
```

Comment:
Just make sure to fix all these kinds of issues

Reply:
...this...